dev_guide/secrets.adoc

```diff
@@ -95,24 +95,37 @@ When creating secrets:
 [[creating-secrets]]
 
 === Creating Secrets
-To create a secret object, use the following command, where the json file is a
+To create a secret object, use the following command, where the JSON file is a
 predefined secret:
 
-====
 ----
 $ oc create -f secret.json
 ----
-====
 
-[[secrets-in-volumes]]
-=== Secrets in Volumes
+[[secrets-in-volumes-and-env-vars]]
+=== Secrets in Volumes and Environment Variables
 
-See xref:secrets-examples[Examples].
+See xref:secrets-examples[examples] of YAML files with secret data.
 
-[[secrets-in-env-vars]]
-=== Secrets in Environment Variables
+After you xref:creating-secrets[create a secret], you can:
 
-See xref:secrets-examples[Examples].
+. Create the pod to reference your secret:
++
+----
+$ oc create -f <your_yaml_file>.yaml
+----
+
+. Get the logs:
++
+----
+$ oc logs <secret_example_pod>
```

Comment:
In the examples given the pods created are actually called "secret-example-pod", you may want to use that rather than <secret_example_pod>